bundles/binding/org.openhab.binding.velux/README.md

```diff
@@ -94,27 +84,16 @@ As the bridge does not support a real rollershutter interaction, this binding pr
 ### Items velux.items
 
 ```
-/**
- * Copyright (c) 2010-2018 by the respective copyright holders.
- *
- * All rights reserved. This program and the accompanying materials
- * are made available under the terms of the Eclipse Public License v1.0
- * which accompanies this distribution, and is available at
- * http://www.eclipse.org/legal/epl-v10.html
- */
-
 /**
  * OpenHAB item defintion for velux binding:
```

Comment:
Are these Javadoc style comments supported in a .items file? If not, they should be converted to the standard // style.